### PR TITLE
Excluding HybridRow and its sub-namespaces

### PR DESCRIPTION
**Pull request by kirankumarkolli**
Excluding HybridRow and its sub-namespaces

### CODE REVIEW

File: xml/_filter.xml [hunk 1]
```diff
@@ -83,6 +83,10 @@
       <typeFilter name="*" expose="false">
       </typeFilter>
     </namespaceFilter>
+    <namespaceFilter name="Microsoft.Azure.Cosmos.Serialization.HybridRow">
+      <typeFilter name="*" expose="false">
```
Comment (kirankumarkolli) [inline xml/_filter.xml]: Will it exclude sub-namespaces also?

**Comment by CamSoper** [comment]
Thank you, but we can't merge this PR. The XMLDocs are generated from the assembly. Please update the source code that generates the XML artifact.

**Comment by CamSoper** [comment]
@kirankumarkolli I'm so sorry about that - I was going through PRs too fast and I totally made the wrong assumption about what this one does.

Thanks to @dend for cleaning up after me! 😄

File: xml/_filter.xml (hunk 1 above)
Comment (scbedd) [reply xml/_filter.xml:87]: It _should_ based on the reading: https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/dotnet/filtering-apis?branch=master

Really though, the easiest way is to simply create a branch on the repo with these changes. I've done that [here](https://github.com/Azure/azure-docs-sdk-dotnet/tree/smoke-test-hybridrowexclude) When the publishing build finishes, we'll be able to review the result.

**Comment by sima-zhu** [comment]
@kirankumarkolli @CamSoper Do we still need the PR? If we do not take any action, I would like to go ahead and close the PR. Thanks!

**Comment by jsquire** [comment]
Thank you for your contribution.  Since there hasn't been recent engagement or a response to Sima's inquiry, I'm going to close this out.  Please feel free to reopen if you'd like to continue working on these changes.

**Pull request closed without merging**